parsigs/parse_sig_api.py

```diff
@@ -78,6 +78,13 @@ class _Frequency:
     times: int
 
 
+@dataclass(frozen=True, eq=True)
```

Comment:
class names in python should not start with an underscore 
And also not between words,
convention is CamelCase